test/decompile/basic.txt

```diff
@@ -119,8 +119,8 @@ import table ns_tab3:funcref;
 table T_b:funcref(min: 0, max: 10);
 export table tab2:funcref(min: 0, max: 11);
 
-data d_a(offset: 0) = "Hello, World!\0a\00";
-data d_b(offset: 100) = 
+data d_HelloWorld(offset: 0) = "Hello, World!\0a\00";
```

Comment:
Should add a data segment that tests the minimum case.

Also a test of the duplicate-identifier behavior.

Reply:
will do